app/helpers/originality_reports_helper.rb

```diff
@@ -19,6 +19,12 @@ def originality_report(score)
     }
   end
 
+  def originality_report_edit_url
+    base_url = tool_proxy_from_assignment.base_tc_url
+    "#{base_url}/api/lti/assignments/#{params['assignment_tc_id']}"\
```

Comment:
It seems unfortunate that we are requiring the tp to know the shape of the url to POST or PUT to.  Have we thought about mechanisms for sending these urls to the TP?